### PR TITLE
Zimwiki.Zim version 0.74.3

### PR DESCRIPTION
**Pull request by celestefox**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/55931)

### CODE REVIEW

File: manifests/z/Zimwiki/Zim/0.74.3/Zimwiki.Zim.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Zimwiki.Zim
+PackageVersion: 0.74.3
+PackageLocale: en-US
+Publisher: Zimwiki
+PackageName: Zim
```
Comment (ItzLevvie) [inline manifests/z/Zimwiki/Zim/0.74.3/Zimwiki.Zim.locale.en-US.yaml:8]:
```suggestion
PackageName: Zim Desktop Wiki
```
This must match Control Panel.

File: manifests/z/Zimwiki/Zim/0.74.3/Zimwiki.Zim.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Zimwiki.Zim
+PackageVersion: 0.74.3
+Installers:
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/z/Zimwiki/Zim/0.74.3/Zimwiki.Zim.installer.yaml:7]:
```suggestion
- Architecture: x64
```

File: manifests/z/Zimwiki/Zim/0.74.3/Zimwiki.Zim.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Zimwiki.Zim
+PackageVersion: 0.74.3
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://zim-wiki.org/downloads/zim-desktop-wiki-0.74.3-setup-w64_x86.exe
+  InstallerSha256: DC9465EA4AD8F194090FFB4A61605ADA80D6BBDCEE24F90C9E0F75D70C6DC51E
```
Comment (ItzLevvie) [inline manifests/z/Zimwiki/Zim/0.74.3/Zimwiki.Zim.installer.yaml:10]:
```suggestion
  InstallerSha256: DC9465EA4AD8F194090FFB4A61605ADA80D6BBDCEE24F90C9E0F75D70C6DC51E
  ProductCode: 404fbece-3a0a-4f4f-b1f1-82ce46af9696
```

File: manifests/z/Zimwiki/Zim/0.74.3/Zimwiki.Zim.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Zimwiki.Zim
+PackageVersion: 0.74.3
+PackageLocale: en-US
+Publisher: Zimwiki
+PackageName: Zim
+License: GPLv2
+ShortDescription: A desktop wiki
```
Comment (ItzLevvie) [inline manifests/z/Zimwiki/Zim/0.74.3/Zimwiki.Zim.locale.en-US.yaml:10]:
```suggestion
ShortDescription: Zim is a graphical text editor designed to maintain a collection of locally stored wiki-pages, a personal wiki.
```

File: manifests/z/Zimwiki/Zim/0.74.3/Zimwiki.Zim.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Zimwiki.Zim
+PackageVersion: 0.74.3
+PackageLocale: en-US
+Publisher: Zimwiki
```
Comment (ItzLevvie) [inline manifests/z/Zimwiki/Zim/0.74.3/Zimwiki.Zim.locale.en-US.yaml:7]:
```suggestion
Publisher: Jaap Karssenberg
```

File: manifests/z/Zimwiki/Zim/0.74.3/Zimwiki.Zim.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Zimwiki.Zim
+PackageVersion: 0.74.3
+PackageLocale: en-US
+Publisher: Zimwiki
+PackageName: Zim
+License: GPLv2
```
Comment (ItzLevvie) [inline manifests/z/Zimwiki/Zim/0.74.3/Zimwiki.Zim.locale.en-US.yaml:9]:
```suggestion
License: GPL-2.0-or-later
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**